api/src/policy/v1beta1.rs

```diff
@@ -0,0 +1,41 @@
+use apimachinery::meta::v1::{DeleteOptions, Metadata, ObjectMeta};
+use apimachinery::meta::{GroupVersion, TypeMeta, TypeMetaImpl};
+use std::borrow::Cow;
+
+const API_GROUP: &str = "policy/v1beta1";
+pub const GROUP_VERSION: GroupVersion = GroupVersion {
+    group: "policy",
+    version: "v1beta1",
+};
+
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Eviction {
+    #[serde(flatten)]
+    typemeta: TypeMetaImpl<Eviction>,
+    #[serde(default)]
+    pub metadata: ObjectMeta,
+    #[serde(default)]
+    pub delete_options: DeleteOptions,
+}
+
+impl TypeMeta for Eviction {
+    fn api_version() -> &'static str {
+        API_GROUP
+    }
+    fn kind() -> &'static str {
+        "Eviction"
+    }
+}
```

Comment:
I'm thinking we want a derive macro for these.

Reply:
I have that macro already, I'm waiting for this PR to be finalised before publishing it.